integrations/langchain/pyproject.toml

```
@@ -11,11 +11,11 @@ requires-python = ">=3.9"
 dependencies = [
     "langchain>=0.2.0",
     "langchain-community>=0.2.0",
-    "mlflow",
     "databricks-vectorsearch>=0.40",
     "databricks-ai-bridge>=0.1.0",
-    "unitycatalog-langchain",
+    "mlflow>=2.20.0",
```

```suggestion
    "mlflow>=2.20.1",
```
Since 2.20.1 has some bugs in agent authoring APIs